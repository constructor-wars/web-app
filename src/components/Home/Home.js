import React from "react";
import "./StyleHome.css";

export default class Home extends React.Component {
  render() {
    return <div>Home component</div>;
  }
}
