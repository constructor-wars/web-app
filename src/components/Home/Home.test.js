import React from "react";
import Home from "./Home";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

describe("Home", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("tests home component", () => {
    const wrapper = shallow(<Home />);
    const text = wrapper.find("div").text();
    expect(text).toEqual("Home component");
  });
});
