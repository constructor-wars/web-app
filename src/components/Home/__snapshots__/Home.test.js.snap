// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home matches the snapshot 1`] = `
<div>
  Home component
</div>
`;
